docusaurus/docs/e2e-test-a-plugin/setup-resources.md

```diff
@@ -0,0 +1,83 @@
+---
+id: setup-resources
+title: Configure the resources you need
```

Comment:
Maybe `Configure any required Grafana resources` or a subset of those words
It doesn't look horrendous even with the longer title. OR we can use `sidebar_label` to have a smaller title in the menu, but full one on the page
<img width="299" alt="Screenshot 2024-02-15 at 17 12 36" src="https://github.com/grafana/plugin-tools/assets/19860021/530ac3de-2045-4f24-9788-4465ecafcbee">

Reply:
Nice - I went with the shorter title for the sidbar_label 👍